fl4health/model_bases/pca.py

```diff
@@ -8,18 +8,46 @@
 from torch.nn.parameter import Parameter
 
 
-class PCAModule(nn.Module):
-    def __init__(self, low_rank: bool = True, full_svd: bool = False, rank_estimation: int = 6) -> None:
+class PcaModule(nn.Module):
+    def __init__(self, low_rank: bool = False, full_svd: bool = False, rank_estimation: int = 6) -> None:
         """
         PyTorch module for performing Principal Component Analysis.
 
         Args:
-            low_rank (bool, optional): Indicates whether the data matrix is low-rank. If the user has
-             prior knowledge that it is, then this parameter can be set to True to allow for more efficient
-              computation of SVD. Defaults to True.
+            low_rank (bool, optional): Indicates whether the data matrix can be well-approximated
+            by a low-rank singular value decomposition. If the user has
+            good reasons to believe so, then this parameter can be set to True to allow for more efficient
+            computations. Defaults to False.
             full_svd (bool, optional): Indicates whether full SVD or reduced SVD is performed. Defaults to False.
```

Comment:
Maybe we can also state that `If low_rank is set to true, this argument is ignored.`?